Sources/CoreDataHero/Extensions/NSPersistentStore+DeleteStoreFiles.swift

```diff
@@ -5,7 +5,7 @@ import CoreData
 extension NSPersistentStore {
     /// Deletes the store's .sqlite, .sqlite-shm, and .sqlite-wal files.
     func deleteStoreFiles() throws {
-        guard let storeURLPath = self.url?.path else {
+        guard let storeURLPath = self.url?.path, !storeURLPath.isEmpty else {
```

Comment:
wonder if there is any benefit to using [file exists apis](https://developer.apple.com/documentation/foundation/filemanager/1410277-fileexists)

Reply:
Good call, I'll look at adjusting these checks!